src/models/api/SavedSearchResponse.ts

```diff
@@ -73,7 +73,7 @@ export function isDateRangeAroundResponse(value: unknown): value is DateRangeAro
 
 export type DateRangeResponse = DateRangeSpanResponse | DateRangeRangeResponse | DateRangePeriodResponse | DateRangeAroundResponse
 
-export function isDateRangeResponse(value: unknown): value is DateRangeRangeResponse {
+export function isDateRangeResponse(value: unknown): value is DateRangeResponse {
```

Comment:
This was the primary issue that caused the types to not error when they should have.